src/Validation/RequestValidator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class RequestValidator implements InstanceValidator {
+
+	use CanValidate;
+
+	public function __construct( MailValidator $mailValidator ) {
+		$this->setConstraints( [
+			new ValidationConstraint( 'email', $mailValidator ),
+			new ValidationConstraint( 'vorname', new RequiredFieldValidator() ),
```

Comment:
Since RequiredFieldValidator does not need any external service or configuration value I'd say it's better to initialize it in RequestValidator. What do you think, @JeroenDeDauw ?